pallets/qf-polkavm/src/mock.rs

```diff
@@ -59,6 +59,7 @@ parameter_types! {
 	pub const PolkaVmMaxStorageKeySize: u32 = 256;
 	pub const PolkaVmMaxStorageSlots: u32 = 4;
 	pub const PolkaVmMinGasPrice: u64 = 1;
+	pub const PolkaVmMinStorageDepositLimit: u64 = 1;
```

Comment:
Why not zero? How does that works with read-only calls?

Reply:
You are right.
Fixed.